examples/common.h

```diff
@@ -17,6 +17,17 @@
 #include <stdlib.h>
 #include <git2.h>
 
+#ifndef PRIuZ
```

Comment:
Arguably, if there was a way to grab our platform code from the main `common.h` file that would be really appreciable when writing examples.

Reply:
Yeah, I had the same thought. Didn't yet want to go down that route, but by using it we could definitely remove some boilerplate code.